src/components/DashboardNav.js

```diff
@@ -1,35 +1,79 @@
-import React, { Component } from "react";
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
 import { NavLink, withRouter } from "react-router-dom";
+import NotificationTip from "../login/components/NotificationTip/NotificationTip";
 import i18n from "../login/translation/InjectIntl_HOC_factory";
 
-class DashboardNav extends Component {
-  render() {
-    return (
-      <nav id="dashboard-nav">
-        <ul>
-          <NavLink exact activeClassName="active" to={`/profile/`}>
-            <li>{this.props.translate("dashboard_nav.profile")}</li>
-          </NavLink>
-          <NavLink
-            exact
-            activeClassName="active"
-            to={`/profile/verify-identity/`}
-          >
-            <li>{this.props.translate("dashboard_nav.identity")}</li>
-          </NavLink>
-          <NavLink activeClassName="active" to={`/profile/settings/`}>
-            <li>{this.props.translate("dashboard_nav.settings")}</li>
-          </NavLink>
-          <NavLink
-            activeClassName="active"
-            to={`/profile/settings/advanced-settings`}
-          >
-            <li>{this.props.translate("dashboard_nav.advanced-settings")}</li>
-          </NavLink>
-        </ul>
-      </nav>
-    );
+function DashboardNav(props) {
+  const [active, setActive] = useState(false);
+  const nins = useSelector(state => state.nins.nins);
+  const phones = useSelector(state => state.phones.phones);
+  const verifiedNin = nins.filter(nin => nin.verified);
+  const verifiedSweNumber = phones.filter(phone => phone.verified && phone.number.includes("+46"));
```

Comment:
Suggest using .startsWith() instead of .includes() here, both for clarity but also for correctness.